Examples/README.md

```diff
@@ -0,0 +1,257 @@
+# MongoDBStitch Examples
+
+This document describes the example applications provided with the [MongoDBStitch library](../README.md).
+
+The following example applications are provided:
+- **DataSender**, it sends data to MongoDB
+- **DataReceiver**, it receives data from MongoDB and confirms the receiving
+
+To see data receiving you need to run **DataReceiver** example after or in parallel with **DataSender** example. We recommend that you:
+- run **DataSender** on the agent of one IMP device
+- run **DataReceiver** on the agent of a second IMP device
+
+Each example is described below. If you wish to try one out, you'll find generic and example-specific [setup instructions](#examples-setup-and-run) further down the page.
+
+## DataSender
+
+This example inserts data into preconfigured MongoDB collection using preconfigured MongoDB Stitch named pipeline:
+
+- Data are sent every 10 seconds.
+- Every data record contains:
+  - `"value"` attribute - integer value, converted to string, which starts at 1 and increases by 1 with every record sent. It restarts from 1 everytime when the example is restarted.
+  - `"measureTime"` attribute - integer value, converted to string, which is the time in seconds since the epoch.
+
+![DataSender example](https://imgur.com/fM75b66.png)
```

Comment:
I'd recommend you store these images locally or in a stable location that is not imgur for longevity purposes.

Reply:
all the images are available locally of course. but I agree with you, we should probably use local references instead